amrit-docker-setup/docker-compose.infra.yml

```
@@ -1,3 +1,7 @@
+# Server/staging infrastructure — env-driven passwords, hardened healthchecks.
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)